### ISSUE TITLE
Implement Travis Build Stages

### ISSUE THREAD

**Issue by LaurentGoderre** [wet-boew/wet-boew#8012]
We need to break down the build

https://blog.travis-ci.com/2017-05-11-introducing-build-stages?utm_source=broadcast&utm_medium=notification

**Comment by nschonni** [comment]
I don't think this is really intended for repos like this that don't have a build matrix.
If they could figure out a way to scale out the Sauce testing after the single build that would be good though.

**Comment by LaurentGoderre** [comment]
What do you mean?

**Comment by nschonni** [comment]
> you can now make one job run only if several other, parallel jobs have completed successfully.

We don't have any parallel jobs that we need to sync before doing a deployment to a service
